src/main/proto/ga4gh/rna_quantification_service.proto

```diff
@@ -2,9 +2,8 @@ syntax = "proto3";
 
 package ga4gh;
 
-import "ga4gh/schemas/ga4gh/rna_quantification.proto";
-import "ga4gh/schemas/ga4gh/common.proto";
```

Comment:
please double check this

Reply:
I think rna_quantification.proto is used here? (message ExpressionLevel) ?

Reply:
![image](https://user-images.githubusercontent.com/18109953/58345867-3f690580-7e0e-11e9-81d9-61141158e502.png)

Yes, only the `common.proto` triggered a warning during the compilation process. The `rna_quantification.proto` is still imported, with a different path. See the screenshot above.